From f55f48c8b49433c309a67191b5b93e43076af35d Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Wed, 5 Sep 2018 11:27:27 -0700
Subject: [PATCH] TRITON-753 imgapi test suite is adding image test files with
 the wrong ownership Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 test/reload-test-data.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/reload-test-data.sh b/test/reload-test-data.sh
index 7097a54..c6679c0 100755
--- a/test/reload-test-data.sh
+++ b/test/reload-test-data.sh
@@ -125,4 +125,6 @@ elif [[ "$opt_mode" == "dc" ]]; then
         echo "icon" >$icon_path
         i=$(($i + 1))
     done
+    # Change image ownership, as imgapi runs as user 'nobody'.
+    chown -R nobody:nobody /data/imgapi/images
 fi
-- 
2.21.0

